lib/wit.js

```diff
@@ -38,11 +40,14 @@ function Wit(opts) {
     ;
   };
 
-  this.converse = (sessionId, message, context) => {
+  this.converse = (sessionId, message, context, reset) => {
     let qs = 'session_id=' + encodeURIComponent(sessionId);
     if (message) {
       qs += '&q=' + encodeURIComponent(message);
     }
+    if (reset) {
```

Comment:
Well, `reset` is part of the `/converse` endpoint, having one function per endpoint makes things simple